queue/queueservice.js

```diff
@@ -291,25 +306,28 @@ class QueueService {
       return base32.encode(h.slice(0, 15)).toString('utf-8').toLowerCase();
     };
 
-    // Construct queue name
-    var name = [
+    // Construct queue name prefix (add priority later)
+    let namePrefix = [
       this.prefix,            // prefix all queues
       hashId(provisionerId),  // hash of provisionerId
       hashId(workerType),     // hash of workerType
-      '1'                     // priority, currently just hardcoded to 1
+      ''                      // priority, add 1 = normal, 5 = high
```

Comment:
Is this just so there is documentation as to what the last value of the string will be?